jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/bpmn2/ProcessDescriptor.java

```diff
@@ -64,6 +66,8 @@
     private Set<String> referencedRules = new HashSet<String>(1);
 
     private Collection<String> signals = Collections.emptySet();
+    private Collection<Signal> signalsMetadata = Collections.emptySet();
+    private Collection<Message> messages = Collections.emptySet();
```

Comment:
This objects are coming from bpm2 parser. They should not be here.
you have an object already for this SignalDesc.
I dont see the counter part MessageDesc